perllib/FixMyStreet/Cobrand/Default.pm

```diff
@@ -658,6 +658,36 @@ sub admin_allow_user {
     return 1 if $user->is_superuser;
 }
 
+=head2 available_permissions
+
+Grouped lists of permission types available for use in the admin
```

Comment:
Perhaps note here which of these permissions are currently in use and which are future? e.g. report_edit_category doesn't appear to be used anywhere.